Sources/Control/Playback.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Playback.swift
+//  ControlKit
+//
+
+import AVFAudio
+import MediaPlayer
+import OSLog
+
+public extension Control {
+    
+    enum Playback {
+        
+        /// Alias for `AVAudioSession.secondaryAudioShouldBeSilencedHint`.
+        public static var isAudioPlaying: Bool { avAudioSession.secondaryAudioShouldBeSilencedHint }
+        
+        private static let avAudioSession = AVAudioSession.sharedInstance()
+        
+        /// Toggles system media playback by activating or disactivating the shared `AVAudioSession`.
+        ///
+        /// Playback that has been paused by this function can normally be resumed if the app playing the content has not been terminated.
+        public static func togglePlayPause() {
+            do {
+                try avAudioSession.setActive(
+                    isAudioPlaying,
+                    options: .notifyOthersOnDeactivation
+                )
+            } catch {
+                log.error("\(error.localizedDescription)")
+            }
+        }
+    }
+}
+
+public extension Control.Playback {
+    
+    enum AppleMusic {
```

Comment:
Seems like it can be marked with MainActor. What is your opinion? 🐼

```suggestion
    @MainActor
    enum AppleMusic {
```